controllers/vrg_volrep_test.go

```diff
@@ -1515,6 +1516,18 @@ func (v *vrgTest) getVRG() *ramendrv1alpha1.VolumeReplicationGroup {
 	return vrgGet(v.vrgNamespacedName())
 }
 
+func updateVRG(vrg *ramendrv1alpha1.VolumeReplicationGroup) {
+	retryErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+		vrgTmp := vrgGet(client.ObjectKeyFromObject(vrg))
+
+		vrgTmp.Spec = vrg.Spec
+
+		return k8sClient.Update(context.TODO(), vrgTmp)
```

Comment:
vrgTmp works, but maybe we can use more meaningful names?

```
func updateVRG(desired *ramendrv1alpha1.VolumeReplicationGroup) {
	retryErr := retry.RetryOnConflict(retry.DefaultBackoff, func() error {
		current := vrgGet(client.ObjectKeyFromObject(desired))

		current.Spec = desired.Spec

		return k8sClient.Update(context.TODO(), current)
```

Reply:
@nirs Done